### ISSUE TITLE
Update Spanish translation for v0.3

### ISSUE THREAD

**Issue by zeel01** [zeel01/monarch#31]
@lozalojo There are a bunch of new strings in the upcoming release.

**Comment by lozanoje** [comment]
> @lozalojo There are a bunch of new strings in the upcoming release.

Perfect, I will check it tomorrow

**Comment by lozanoje** [comment]
Done! https://github.com/zeel01/monarch/pull/33

What are the toggleMarkers for? I have not seen anything related in the module

**Comment by zeel01** [comment]
In the new version, you can right-click on the cards for a context menu. This menu gives you a bunch of colored "markers" to select to place marks on your cards. It's a very generic system to allow users to better organize. Those strings are for tool tips on the markers and marker toggle buttons.

**Comment by lozanoje** [comment]
Oh, that's the feature we spoke (i'm Viriato139ac at discord and also lozalojo at github, I switched to this account for Foundry related matters) when we discussed about grouping cards to support Torg.

I made a quick fix in torg system changing the flag to monarch.markers.purple and now pooled works, I've posted a video at discord Torg channel. I will ask the dev to make something more elaborated (check if monarch is enabled, populate legacy to monarch flags and back in case monarch is not present).
